package-lock.json

```diff
@@ -74,9 +74,9 @@
       }
```

Comment:
should this file be committed?

Reply:
@willc-work did you mean to commit this file?

Reply:
think it was added in error, replaced it with the version from main